Makefile

```diff
@@ -32,8 +36,12 @@ format-go:
 format-shell:
 	shfmt -w scripts/*.sh
 
+.PHONY: format-solidity
+format-solidity:
+	"$$(npm config get prefix)/bin/prettier" --print-width 100 --write $$(find ethereum -name '*.sol')
```

Comment:
The default width is 80.  We are using 120 for Go code. I tried 120, but I thought Solidity was a little easier to read at 100 than 120.